asm/macros/battle_script.inc

```diff
@@ -1061,13 +1061,21 @@
 	setsemiinvulnerablebit TRUE
 	.endm
 
-	.macro jumpifweathercheckchargeeffects battler:req, checkChargeTurnEffects:req, jumpInstr:req
+	.macro tryfiretwoturnmovenowcheckeffects battler:req, checkChargeTurnEffects:req, jumpInstr:req
```

Comment:
Maybe a different name? I understand the name change but there is a lot going on in `try fire two turn move now check effects`

Reply:
Well what do you suggest?

Reply:
maybe just `tryfiretwoturnmovenow` with check effects?

Reply:
Hmm, I suppose since that function isn't directly called from anywhere in BS1 anymore there's no need to hint what the second arg does. @Bassoonian `tryfiretwoturnmovenow` good with you?

Reply:
Fine by me, yeah! I feel like intuitively I'd expect Power Herb to be checked there too, though, so maybe `tryfiretwoturnmovenowcontext` or something? Although that gets unnecessarily long, too, so I'm fine with Alex's suggestion

Reply:
Hmm, then perhaps a better suggestion would be `tryfiretwoturnmovenowbyeffect` if we want to make It clear that it doesn't include Power Herb (which it definitely shouldn't)? Otherwise I'd just go with Alex's suggestion since at least you can tell from the script that the Power Herb check is supposed to be separate.

Reply:
I think both are an improvement over what's currently in the PR, and I don't have a preference for either of the suggestions :)

Reply:
@AlexOn1ine Then it's up to the one with special powers :)

Reply:
`tryfiretwoturnmovenowbyeffect` is fine